src/main/java/org/alliancegenome/curation_api/model/ingest/dto/fms/BioSampleGenomicInformationDTO.java

```diff
@@ -0,0 +1,13 @@
+package org.alliancegenome.curation_api.model.ingest.dto.fms;
+import org.alliancegenome.curation_api.model.ingest.dto.base.BaseDTO;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class BioSampleGenomicInformationDTO extends BaseDTO{
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'{' is not preceded with whitespace.
<!-- __reviewdog__:ChAyODUwYzI3MDA4YTg5YmM4EgpjaGVja3N0eWxl -->